internal/store/build_result.go

```diff
@@ -21,26 +21,38 @@ type BuildResult struct {
 	// The tag is derived from a content-addressable digest.
 	Image reference.NamedTagged
 
-	// If this build was a container build, containerIDs we built on top of
-	ContainerIDs []container.ID
+	// When we `docker-compose up` a service, we wait synchronously for the container to start.
```

Comment:
this and the comment below don't describe WHAT the field represents, which imo is more helpful.

> The ID of the docker compose container we deployed to (when we `docker-compose up`...)

> IDs of containers we LiveUpdated (i.e. containers whose contents have diverged...)

Reply:
done